recipes/omniscidb-cuda/recipe/meta.yaml

```diff
@@ -0,0 +1,105 @@
+{% set name = "omniscidb-cuda" %}
```

Comment:
One more question. Why add `-cuda`?

Reply:
Because there is also `omniscidb-cpu`.

Reply:
You should use one feedstock for both to avoid duplication, but this can be done later.

Reply:
Yes, I agree, we'll continue working on this.